### PR TITLE
Add hexside_length for newer tmx.

### PR DESCRIPTION
**Pull request by darkautism**
Some of tmx (new version?) file has this attarbute. And i found other tmx library also has this attribute. We need this!

### CODE REVIEW

File: src/map.rs [hunk 1]
```diff
@@ -48,6 +48,8 @@ pub struct Map {
     /// individual tiles may have different sizes. As such, there is no guarantee that this value
     /// will be the same as the one from the tilesets the map is using.
     pub tile_height: u32,
+    /// The Length of the side of a hexagonal tile (used by tile layers on hexagonal maps).
```
Comment (aleokdev) [inline src/map.rs]:
```suggestion
    /// The length of the side of a hexagonal tile in pixels (used by tile layers on hexagonal maps).
```

**Review by aleokdev — COMMENTED** [review]
Thank you! Just one small thing...

**Comment by aleokdev** [comment]
Please also update CHANGELOG.md and select the `next` branch as the merge target.

**Comment by darkautism** [comment]
I didn't know which should been in the CHANGELOG.md, so I cannot fix this conflicts .

**Comment by aleokdev** [comment]
Alright, I'll fix it

**Comment by aleokdev** [comment]
Had to rebase your branch to merge into `next`, please for next time base everything onto the `next` branch instead

**Review by aleokdev — DISMISSED** [review]

**Comment by aleokdev** [comment]
GH Actions froze because of the CI skip on the previous commit lol

**Review by aleokdev — DISMISSED** [review]

File: src/map.rs [hunk 2]
```diff
@@ -56,6 +56,8 @@ pub struct Map {
     /// individual tiles may have different sizes. As such, there is no guarantee that this value
     /// will be the same as the one from the tilesets the map is using.
     pub tile_height: u32,
+    /// The length of the side of a hexagonal tile in pixels (Used by tile layers on hexagonal maps).
+    pub hexside_length: Option<u32>,
```
Comment (bjorn) [inline src/map.rs]: I'm guessing it should really be called `hex_side_length`, since "hexside" is not a word.

File: src/map.rs [hunk 3]
```diff
@@ -56,6 +56,8 @@ pub struct Map {
     /// individual tiles may have different sizes. As such, there is no guarantee that this value
     /// will be the same as the one from the tilesets the map is using.
     pub tile_height: u32,
+    /// The length of the side of a hexagonal tile in pixels (Used by tile layers on hexagonal maps).
```
Comment (bjorn) [inline src/map.rs]:
```suggestion
    /// The length of the side of a hexagonal tile in pixels (used by tile layers on hexagonal maps).
```

**Review by bjorn — COMMENTED** [review]
Looks good, though I've suggested some small tweaks. Please remember to squash when merging. :-)

**Review by aleokdev — APPROVED** [review]

**Comment by bjorn** [comment]
Whoops, I missed that a `u32` was used. Actually this property isn't unsigned, even if there may not be an actual use for negative values... (opened https://github.com/mapeditor/rs-tiled/pull/316).

**Pull request merged**